js/treetest.js

```diff
@@ -109,14 +121,97 @@ var tasks = {
 		this.idx = number;
 		$('#taskDesc').html(this.list[number]);
 		$('#taskNum').html("Task "+(number+1)+" of "+this.list.length);
+
+		// Resetting the tree between tasks results in all open nodes closing, but
+		// we don't want to register these as close node events, because they aren't
+		// meaningfully user actions. So we pause listening for close node events
```

Comment:
When the task changes, the tree gets reset. This includes at page load when the active task is set to the first task. So even if there is only one task, this will still happen once. It is important not to accidentally emit 'close node' events just because the tree is being set up.